packages/components/src/segmented-control/stories/index.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	alignCenter,
+	alignJustify,
+	alignLeft,
+	alignRight,
+} from '@wordpress/icons';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Icon, __experimentalSpacer as Spacer } from '../../';
+import { SegmentedControl } from '../index';
+import { View } from '../../view';
+
+export default {
+	component: SegmentedControl,
+	title: 'Components/SegmentedControl',
+};
+
+const align = [
+	{
+		label: <Icon icon={ alignLeft } size={ 14 } />,
+		value: 'left',
+	},
+	{
+		label: <Icon icon={ alignCenter } size={ 14 } />,
+		value: 'center',
+	},
+	{
+		label: <Icon icon={ alignRight } size={ 14 } />,
+		value: 'right',
+	},
+	{
+		label: <Icon icon={ alignJustify } size={ 14 } />,
+		value: 'justify',
+	},
+];
```

Comment:
Currently, the icons used in the story don't show when are highlighted (black `fill` color over a black background)

<img width="807" alt="Screenshot 2021-07-08 at 12 15 40" src="https://user-images.githubusercontent.com/1083581/124905329-38011800-dfe6-11eb-9f27-aed5b7410546.png">

One quick way to solve this is to apply the CSS rule `fill: currentColor` to all `<Icon />`s

```suggestion
const commonAlignIconProps = {
	size: 14,
	style: {
		fill: 'currentColor',
	},
};

const align = [
	{
		label: <Icon icon={ alignLeft } { ...commonAlignIconProps } />,
		value: 'left',
	},
	{
		label: <Icon icon={ alignCenter } { ...commonAlignIconProps } />,
		value: 'center',
	},
	{
		label: <Icon icon={ alignRight } { ...commonAlignIconProps } />,
		value: 'right',
	},
	{
		label: <Icon icon={ alignJustify } { ...commonAlignIconProps } />,
		value: 'justify',
	},
];
```